widgets/case-feedback-form/src/Components/Footer.vue

```diff
@@ -0,0 +1,63 @@
+<!--
+<COPYRIGHT>
+
+    Copyright © 2016-2025, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+-->
+<script setup>
+import { defineProps } from 'vue';
+
+const props = defineProps({
+    logo: {
+        type: String,
+        required: true,
+    },
+});
+</script>
+
+<template>
+    <div class="flex w-full flex-col">
+        <div class="mt-4 flex w-full justify-center">
+            <img class="block h-5" :src="logo" alt="Advising App Logo" />
+        </div>
+
+        <div class="flex w-full justify-center pb-4 pt-2">
+            <span class="w-11/12 text-center text-xs lg:w-3/4 xl:w-7/12">
+                © 2016-{{ new Date().getFullYear() }} Canyon GBS LLC. All Rights Reserved. Canyon GBS™ and Advising
+                App™ are trademarks of Canyon GBS LLC. For more information or inquiries, please visit our website at
```

Comment:
Instead of ™ for Canyon GBS and Advising App, these need to be changed to the registered trademark ®